File: exercises/101-109_small_problems/easy_01/go/easy_02.go
package main

import "fmt"

func is_odd(num int) bool  {
  return num % 2 != 0
}

func main() {
  fmt.Println(is_odd(2) == false)
  fmt.Println(is_odd(5) == true)
  fmt.Println(is_odd(-17) == true)
  fmt.Println(is_odd(-8) == false)
  fmt.Println(is_odd(0) == false)
  fmt.Println(is_odd(7) == true)
}

File: exercises/101-109_small_problems/easy_01/rust/easy_02.rs
fn main() {
    // All of these should print "true"
    println!("{}", is_odd(2) == false);
    println!("{}", is_odd(5) == true);
    println!("{}", is_odd(-17) == true);
    println!("{}", is_odd(-8) == false);
    println!("{}", is_odd(0) == false);
    println!("{}", is_odd(7) == true);
}

// easy_02
fn is_odd(num: i32) -> bool {
    (num % 2) != 0
}
